CHANGELOG.md

```diff
@@ -4,6 +4,28 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v4.2.1](https://github.com/voxpupuli/puppet-yum/tree/v4.2.1) (2020-07-19)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-yum/compare/v4.2.0...v4.2.1)
+
+**Implemented enhancements:**
+
```

Comment:
since this changelog contains enhancements, it should be a 4.3.0 release, not 4.2.1

Reply:
Updated